scss/civicrm/contact/pages/_events.scss

```diff
@@ -1,3 +1,16 @@
+.page-civicrm-event-search {
+  [class*=CRM_Event_Form_Task_] {
+    .crm-form-block > .crm-submit-buttons > .crm-button:first-of-type {
+      margin-left: 0;
+    }
+
+    .status {
+      background-color: $crm-light-green-color;
+      border: 0;
```

Comment:
Why we need to have border: 0; can't this be like alert component like in style guide?